pkg/utils/maps/maps.go

```diff
@@ -19,6 +19,16 @@ func IsSubset(toCheck, fullSet map[string]string) bool {
 	return true
 }
 
+// ManagedKeysChanged compares the values of the given keys managed by the elasticsearch operator between two maps and returns true if they are not equal.
+func ManagedKeysChanged(map1, map2 map[string]string, managedkeysToCompare []string) bool {
```

Comment:
```suggestion
func IsEqualSubset(map1, map2 map[string]string, keySubSet []string) bool {
```
I think there is nothing "managed" about this on the level of the map util package.

Reply:
Thanks, I was not happy with the name myself.